sdk/schemaregistry/Azure.Data.SchemaRegistry/README.md

```diff
@@ -90,7 +90,7 @@ Register a schema to be stored in the Azure Schema Registry. When registering a
 
 ```C# Snippet:SchemaRegistryRegisterSchema
 string name = "employeeSample";
-SerializationType type = SerializationType.Avro;
+SchemaFormat format = SchemaFormat.Avro;
```

Comment:
I still see mentions of serialization type & content in the readme outside the code snippets.